l5kit/l5kit/rasterization/box_rasterizer.py

```diff
@@ -137,11 +141,11 @@ def rasterize(
                     im = draw_boxes(self.raster_size, world_to_image_space, np.append(agents, av_agent), 255)
```

Comment:
Can we make this name a bit more descriptive, please? Something like `rasterized_agent_boxes_frame`

`im_ego` => rasterized_ego_box_frame

Or something, it's a bit confusing to read now and understand what is im, agents_im, ego_im.